website/docs/docs/dbt-versions/cloud-release-tracks.md

```diff
@@ -15,10 +15,10 @@ By moving your environments and jobs to release tracks you can get all the funct
 ## Which release tracks are available?
 
 - **"Latest"** (available to all plans, formerly called "Versionless"): Provides a continuous release of the latest functionality in dbt Cloud. Includes early access to new features of the dbt framework before they're available in open source releases of dbt Core.
```

Comment:
might benefit having this turn into tables so it's easier to read - happy to do that for you @jtcohen6 if you have no issues with that?

Reply:
I'll take a crack at this!

Reply:
No probs and this is the markdown to help:
```

| Release track | Description | Availability | 
| ------------- | ------------ | ---------- | 
| **"Latest"** |  Formerly called "Versionless". Provides a continuous release of the latest functionality in dbt Cloud. Includes early access to new features of the dbt framework before they're available in open source releases of dbt Core. | Available to all plans. |
| **"Compatible"** `<Lifecycle status="preview"/>`  | Provides a monthly release aligned with the most recent open source versions of dbt Core and adapters, plus functionality exclusively available in dbt Cloud. |  Available to Team + Enterprise |
| **"Extended"** `<Lifecycle status="preview"/>` | The previous month's "Compatible" release. | Available to Enterprise |

```

Reply:
No probs and this is the markdown to help:
```

| Release track | Description | Availability | 
| ------------- | ------------ | ---------- | 
| **"Latest"** |  Formerly called "Versionless". Provides a continuous release of the latest functionality in dbt Cloud. Includes early access to new features of the dbt framework before they're available in open source releases of dbt Core. | Available to all plans. |
| **"Compatible"** `<Lifecycle status="preview"/>`  | Provides a monthly release aligned with the most recent open source versions of dbt Core and adapters, plus functionality exclusively available in dbt Cloud. |  Available to Team + Enterprise |
| **"Extended"** `<Lifecycle status="preview"/>` | The previous month's "Compatible" release. | Available to Enterprise |

```